app/models/access_request.rb

```diff
@@ -0,0 +1,51 @@
+# == Schema Information
+#
+# Table name: access_request
+#
+#  id               :integer          not null, primary key
+#  email_address    :text
+#  first_name       :text
+#  last_name        :text
+#  organisation     :text
+#  reason           :text
+#  request_date_utc :datetime         not null
+#  requester_id     :integer
+#  status           :integer          not null
+#  requester_email  :text
+#
+
+class AccessRequest < ApplicationRecord
+  belongs_to :user, foreign_key: :requester_id, primary_key: :id
+
+  enum status: {
+    requested: 0,
```

Comment:
You don't need to map to the integers, it happens by default. I think the recommendation is to not do it unless you need to.